shell/src/test/java/tachyon/shell/TfsShellTest.java

```diff
@@ -84,6 +92,8 @@ public final void before() throws Exception {
     mNewOutput = new PrintStream(mOutput);
     mOldOutput = System.out;
     System.setOut(mNewOutput);
+    // clear the loginUser
+    Whitebox.setInternalState(LoginUser.class, "sLoginUser", (String) null);
```

Comment:
Is it necessary to set this before all tests? Could it be placed within the tests that need it?